modules/datadict/php/datadict.class.inc

```diff
@@ -97,8 +97,17 @@ class Datadict extends \DataFrameworkMenu
             }
             $dictInstruments[$key] = $otherNames . $name;
         }
+        // get cohorts
+        $user     = \NDB_Factory::singleton()->user();
+        $projects = $user->getProjectIDs();
+        $cohorts  = [];
+        foreach ($projects AS $projectID) {
+            $cohorts = $cohorts + \Utility::getCohortList($projectID);
+        }
+
```

Comment:
this code section is already repeated twice in this PR. we should find a way to reduce duplication